tools-integration/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id("com.hedera.hashgraph.java-conventions")
+    id("org.openjfx.javafxplugin") version "0.0.13"
+}
+
+dependencies {
+//    implementation("pl.pragmatists:JUnitParams:1.1.1")
+    testImplementation(project(":tools-core"))
+    testImplementation(project(":tools-ui"))
+    testImplementation(project(":tools-cli"))
+    testImplementation(libs.bundles.logging)
+    testImplementation(libs.commons.io)
+    testImplementation(libs.gson)
+    testImplementation(libs.controlsfx)
+    testImplementation(libs.opencsv)
+    testImplementation(testLibs.testfx.core)
+    testImplementation(testLibs.testfx.junit)
```

Comment:
I assume that testfx.junit depends on testfx.core?

Reply:
I thought so, based off all the posts and documentation I found on it. But doing some more digging, I see some say it does not, and testfx.core can be left out. I'll keep digging.